docs/antora.yml

```diff
@@ -0,0 +1,44 @@
+name: jekyll-asciidoc
```

Comment:
I think the component name should be jekyll-integration (or jekyll-int for short). WDYT?